src/main/webapp/script.js

```diff
@@ -37,11 +37,12 @@ function closeRemoveMemberForm() {
 }
 
 function loadFamilyMembers() {
-  fetch('/family').then(response => response.json()).then((memberEmails) => {
+  fetch('/family').then(response => response.json()).then((family) => {
     const familyElement = document.getElementById('family-container');
     const familyHeader = document.createElement("HEADER");
-    familyHeader.innerText = "Current Family Members: ";
+    familyHeader.innerText = "Current Family Members in " + family.name + ":";
```

Comment:
Can you verify that this is not vulnerable to XSS attacks?

(e.g. if someone tries to add a family named `<b>Foo</b>`, it will show the text and *not* bold). That way, no one can inject scripts into your code)